install/aws-terraform/modules/registry/templates/regenerate-ecr.tpl

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# Set the AWS auth environment variables
+AWS_DEFAULT_REGION=${region}
+AWS_SECRET_ACCESS_KEY=${secret_key}
+AWS_ACCESS_KEY_ID=${access_key}
```

Comment:
These needs some `export` in order to be available in the TOKEN subshell below. With that said though, I believe it is much better if those ENV are set on the cronjob at runtime and are proper Kubernetes secrets instead. Embedding them like this in the command makes them readable to most people with access to Kubernetes as you would like to allow people to see cronjobs but NOT secrets.